modules/cluster-entitlements.adoc

```diff
@@ -39,17 +39,19 @@ ifndef::openshift-origin[]
 
 In {product-title} {product-version}, you require access to the internet to
 ifndef::restricted[]
-install and
+install
 endif::restricted[]
 ifdef::restricted[]
-obtain the images that you require to install the cluster and to
```

Comment:
@openshift/team-documentation Please also check the updated wording for ../installing/installing_vsphere/installing-restricted-networks-vsphere.html and ../installing/installing_bare_metal/installing-restricted-networks-bare-metal.html. A different way of saying this (to remove the duplicate "you require") could be "... you require access to the internet to obtain the images that are necessary to install your cluster."

Reply:
@lbarbeevargas, I'm good with either.